README.md

```diff
@@ -1,28 +1,38 @@
-# DKAN Tools (DKTL)
+# DKAN Tools
 
 This CLI application provides tools for implementing and developing [DKAN](https://getdkan.org/), the Drupal-based open data portal.
 
 ## Requirements
 
-This tool currently only supports [Docker](https://www.docker.com/)-based local development environments. In the future it will be expanded to support a local webserver and database setup. Current requirements are simply:
+DKAN Tools was designed with a [Docker](https://www.docker.com/)-based local development environment in mind. Current requirements are simply:
 
 * Bash-like shell that can execute .sh files (Linux or OS X terminals should all work)
 * [Docker](https://www.docker.com/get-docker)
 * [Docker Compose](https://docs.docker.com/compose/)
 
 That's it! All other dependencies are included in the Docker containers that dkan-tools will create.
 
+It is also possible to run most DKAN Tools commands using a local webserver, database and PHP setup, but this practice is less supported.
+
 ## Installation
 
 1. Download or clone this repository into any location on your development machine.
-2. Add _/bin/dktl_ to your path somehow. This is often accomplished by adding a symbolic link to a folder already in your path, like _~/bin_. For instance: 
+2. Add _bin/dktl_ to your `$PATH` somehow. This is often accomplished by adding a symbolic link to a folder already in your path, like _~/bin_. For instance: 
 
 ```bash
 ln -s /my/dktl/location/bin/dktl ~/bin/dktl
```

Comment:
that 'location' bit might be confusing to the literal-minded, how about `ln -s /myworkspacelocation/dktl/bin/dktl ~/bin/dktl